cmd/admin-handlers.go

```diff
@@ -2908,6 +2972,10 @@ func (a adminAPIHandlers) InspectDataHandler(w http.ResponseWriter, r *http.Requ
 	if !errors.Is(err, errFileNotFound) {
 		logger.LogIf(ctx, err)
 	}
+
+	dataZipW.Close()
+	encw.Close()
```

Comment:
Maybe log an error here. If the encrypted stream cannot be closed properly, it cannot be decrypted since `Close` writes any remaining final encrypted chunk.

Reply:
Done